.github/workflows/ci.yml

```
@@ -159,10 +159,73 @@ jobs:
         run: rush install
       - name: Build
         run: rush build -t sample-static-html-composites
-      - name: Tests
-        run: rush test -o sample-static-html-composites
+      - name: Automation Tests
```

nit: can we name this task visual regression tests
```suggestion
      - name: Visual Regression Tests
```